app/controllers/idv/by_mail/request_letter_controller.rb

```diff
@@ -11,11 +11,11 @@ class GpoController < ApplicationController
     before_action :confirm_profile_not_too_old
 
     def index
-      @presenter = GpoPresenter.new(current_user, url_options)
+      @presenter = Idv::GpoPresenter.new(current_user, url_options)
```

Comment:
I haven't done a lot with presenters, but I think they usually match the controller name.
```suggestion
      @presenter = Idv::RequestLetterPresenter.new(current_user, url_options)
```